crates/wallet/src/wallet/tx_builder.rs

```diff
@@ -763,16 +766,38 @@ impl fmt::Display for AddForeignUtxoError {
 #[cfg(feature = "std")]
 impl std::error::Error for AddForeignUtxoError {}
 
+type TxVecSort<T> = dyn Fn(&T, &T) -> std::cmp::Ordering;
+
```

Comment:
Not sure the caller needs to know about the internal representation. I think `TxSort` is fine

Reply:
I wanted to avoid ambiguity. I don't know if there is any valid application for some `TxSort` actually sorting whole transactions.

Confirm me here and I will change it.

Reply:
> actually sorting whole transactions

Can you elaborate on this?

Reply:
I think what @rustaceanrob is saying is that you don't need to mention `Vec` in the name of this type, it could just be `TxSort<T>`. It's a small thing but I agree there's not need to mention Vec in the name.

Reply:
> > actually sorting whole transactions
> 
> Can you elaborate on this?

It's more of a naming approach, like _if you are specific enough, you don't risk to overlap with future changes_ that may require similar names. As both of you don't see it plausible, and I'm not strongly convinced about those future overlapping names, I agree to change it.